docfx_project/articles/bootstrapping-trust/part-01.encryption-everywhere.md

```diff
@@ -0,0 +1,310 @@
+fulfilled, a session is associated with the client and service. This new
+session is propagated to the necessary Ziti Routers, and the required
+circuits are created. The requesting client is returned a list of Ziti
+Routers that can be connected to complete the last mile of communication
+between the Ziti Overlay Network and the SDK client.
+
+This set of steps covers the pillars of the Zero Trust model! The Ziti
+Controller and SDK Clients verify each other. The client cannot connect
+to network resources or services until it authenticates. After
+authentication, a client is given the least privilege access allowed by
+only being told about and only being able to dial/bind assigned
+services. It is a Zero Trust overly network!
+
+How did this system come into existence? How do the Ziti SDK client and
+Ziti Controller verify each other? How do the routers and controller
+know to validate each other? How is this managed at scale with hundreds
+o Ziti Routers and thousands of Ziti SDK clients? It seems that this is
+a recursive problem and to terminate the recursion we have to start our
+system with a well defined and carefully controlled seed of trust.
+
+# Trust
+
+In software systems that require network connectivity, there are at
+least two parties in the system. Generally, there are more and in the
+case of a Ziti network, there could be thousands. Between these parties,
+each time a connection is made, a trust decision is made. Should this
+connection be allowed? Mechanisms must be put into place to verify the
+identity of the connecting party.
+
+One mechanism that might jump out at the reader is a password or secret.
+In Ziti it would be possible to configure the Controller, Routers, and
+SDK Clients with a secret. Software is easy to deploy with a secret.
+Throw it into a configuration file, point the software at, and off you
+go!
+
+It is also fundamentally weak as there is only one secret in the system
+necessary to compromise the entire system. In Ziti, this would mean
+giving the secret to network clients that may or may not be owned by the
+network operator. Also, shared secrets do not individually identify each
+component, nor do they define how secrets will power other security
+concerns, like encryption.
+
+The solution can be improved. Secrets could be generated per software
+component. The controller, each router, and each SDK client could have a
+unique secret. This secret would then individually identity each
+component! It is a significant improvement, but how does each component
+verify connections? Do they challenge for the incoming connections
+secret and compare it to a list? That means that a pair of systems that
+need to connect must have each other's secrets. Secret sharing will not
+do! We can not be copying secrets between every machine. One machine
+that is compromised would mean that many secrets will are revealed!
+
+This solution can be evolved and improved, but we do not have to it!
```

Comment:
"but we do not have to it!" 

what don't we have to "it"?  :D

Reply:
Re-worded paragraph.